changelog/2663.feature.rst

```diff
@@ -0,0 +1,2 @@
+Using MetaClasses to enable at tab completion of Attrs defined dynamically.
```

Comment:
I would re-write this to be more user focused.

"Net search attributes now support tab completion of values and display a table of possible values when printed, to allow easier discoverability of possible search values." or similar